File: src/tests/Markify.Fixtures/Projects/Source/Interface/InterfaceGenerics.cs
using System.Collections.Generic;

public interface SingleGenericType<T> { }

public interface MultipleGenericType<T, Y> 
    where T : class, IList<string>
    where Y : struct{ }

public interface CovariantGenericType<in T> { }

public interface ContravariantGenericType<out T> { }

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Interface/InterfaceGenerics.vb
Imports System.Collections.Generic

Public Interface SingleGenericType(Of T)
End Interface

Public Interface MultipleGenericType(Of T As {Class, IList(Of String)}, Y As Structure)
End Interface

Public Interface CovariantGenericType(Of In T)
End Interface

Public Interface ContravariantGenericType(Of Out T)
End Interface